.builds/freebsd.yml

```diff
@@ -32,7 +33,7 @@ tasks:
     cd subprojects
     ln -s ../../wlroots wlroots
     cd ..
-    meson build
+    meson build -Dtray=enabled -Dsd-bus-provider=basu
```

Comment:
Can't use `-Dauto_features=enabled` instead of `-Dtray=enabled` due to https://github.com/mesonbuild/meson/issues/5320.